ThumbnailItem/ThumbnailItem.js

```diff
@@ -40,25 +99,48 @@ const ThumbnailItemBase = kind({
 		publicClassNames: true
 	},
 
-	render: ({children, css, src, ...rest}) => {
+	computed: {
+		className: ({roundThumbnail, styler}) => styler.append({
+			roundThumbnail: roundThumbnail === true
```

Comment:
You can simply write "styler.append({ roundThumbnail})" when names are the same (and simplify condition "roundThumbnail === true" by just passing roundThumbnail).

Reply:
Thanks for suggestion :)